apps/camera/style/settings.css

```diff
@@ -145,7 +145,7 @@
 .settings_header {
   position: relative;
   height: 9rem;
-  margin-right: 5rem; /* 1 */
+  -moz-margin-end: 5rem; /* 1 */
```

Comment:
This line broke the header alignment. The margin is used to indent the text from the blue settings button. This button *doesn't* change position in RTL mode.

NOTE: Please check the header alignment looks correct in all four rotations :)